### ISSUE TITLE
Rework the documentation of the classes.

### ISSUE THREAD

**Issue by SeppPenner** [Joelius300/ChartJSBlazor#12]
The documentation of the classes as well as the code style is not consistent. In my opinion, it would make sense to re-work this as well when reworking the different charts. (https://github.com/Joelius300/ChartJSBlazor/projects/1)

**Comment by SeppPenner** [comment]
I would like to help you with this library. What needs to be done first?

**Comment by Joelius300** [comment]
Do you mean the XML-documentations in the Code? Or what documentations are you talking about?

**Comment by SeppPenner** [comment]
Yeah, I mean the XML doc comments in this case. They are there sometimes and sometimes not. Most of the time, they're inconsistent. (Or did you already change something?)

The README needs to be updated, too but this does only make sense if you/ we have enough examples for all chart types. This is what https://github.com/Joelius300/ChartJSBlazor/issues/15 is about, I guess.

**Comment by Joelius300** [comment]
This is one of the things I'm working on currently. See [this commit](https://github.com/Joelius300/ChartJSBlazor/commit/20d3acab3b580618b8d9839152c935882edaa051).  
Also it's worth mentioning that I created a file [XML_Conventions.md](https://github.com/Joelius300/ChartJSBlazor/blob/master/XML_Conventions.md) where I put some "rules" for this project so the XML-docs can stay consistent.  
I updated `LineChart` and related classes after these conventions and I think they're not bad (but please tell me what you'd change if you see anything). 

I will do the others while reworking the charts.  

And yes that's exactly what https://github.com/Joelius300/ChartJSBlazor/issues/15 is for :)

**Comment by Joelius300** [comment]
Closing this for now since I don't think it's still necessary. If you want to change something specific about the XML-docs (and or conventions) you should open a new issue or a PR directly :)

**Comment by SeppPenner** [comment]
> This is one of the things I'm working on currently. See this commit.
Also it's worth mentioning that I created a file XML_Conventions.md where I put some "rules" for this project so the XML-docs can stay consistent.

Looks good :) The XML_Conventions file is a good idea that everyone can see what comment style they should use.
